app/grandchallenge/pages/urls.py

```diff
@@ -15,6 +16,7 @@
     path("pages/", PageList.as_view(), name="list"),
     path("pages/create/", PageCreate.as_view(), name="create"),
     path("", ChallengeHome.as_view(), name="home"),
+    path("/admin", ChallengeSettingsBase.as_view(), name="admin"),
```

Comment:
Paths should not start with a `/`, but end with a `/`
```suggestion
    path("admin/", ChallengeSettingsBase.as_view(), name="admin"),
```